src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+function getFilteredGender(people, gender) {
+  return people.filter(genderCheck => genderCheck.sex === gender);
+}
+
+function getAverageAge(people) {
+  const TotalAge = people.map(person => person.died - person.born);
+  const AverageAge = TotalAge.reduce((total, current) =>
+    total + current) / TotalAge.length;
+
+  return AverageAge;
```

Comment:
these are not classes, variable names should start from lowercase letter
```suggestion
  const totalAge = people.map(person => person.died - person.born);
  const averageAge = TotalAge.reduce((total, current) =>
    total + current) / TotalAge.length;

  return averageAge;
```